packages/react-server/src/ReactFizzServer.js

```diff
@@ -1580,6 +1601,13 @@ function finishClassComponent(
     nextChildren = instance.render();
   }
 
+  if (request.status === ABORTING) {
```

Comment:
There are so many places we need to check this for Fizz so I think maybe we should just put it after everywhere we call retryNode:

https://github.com/facebook/react/blob/main/packages/react-server/src/ReactFizzServer.js#L2516

Or maybe at least before every return inside retryNode that might have user space code in it.

Reply:
discussed offline. This is tricky b/c we can end up aborting too high. For now we'll just support the obvious places where user code can run like function and class components and lazy initializers. We can revisit and expand support to iterables and other places if we find a strong motivation in the future. The use case for aborting in these other places is a bit esoteric